src/PostgREST/MediaType.hs

```diff
@@ -19,7 +19,9 @@ import Protolude
 -- | Enumeration of currently supported media types
 data MediaType
   = MTApplicationJSON
+  | MTArrayJSONStrip
   | MTSingularJSON
+  | MTSingularJSONStrip
```

Comment:
How about turning the `nulls=stripped` to a parameter of the existing media types?

```suggestion
  | MTSingularJSON Bool
```

Reply:
Yes, that is what I thought at first, but we would still need `MTArrayJSONStrip` for `Accept: application/vnd.pgrst.array+json;nulls=stripped`. Additionally, with `MTSingularJSON Bool`, we would have to make changes at all the places where `MTSingularJSON` is used, for example in `OpenAPI.hs` which I don't understand yet:
```haskell
./Response/OpenAPI.hs:353:  & produces ?~ makeMimeList [MTApplicationJSON, MTSingularJSON <True/False>]
./Response/OpenAPI.hs:409:  & produces .~ makeMimeList [MTApplicationJSON, MTSingularJSON <True/False>, MTTextCSV]
./Response/OpenAPI.hs:410:  & consumes .~ makeMimeList [MTApplicationJSON, MTSingularJSON <True/False>, MTTextCSV]
```
However, still I think it is much simpler. Let me push the changes.